src/services/conversionRates/getConversionRatesService.js

```diff
@@ -208,7 +208,7 @@ const getHourlyUSDRateCryptocompare = async (timestamp, tokenSymbol) => {
     ),
   );
 
-  const tsData = resp && resp.Data && resp.Data.find(d => d.time === timestampMS);
+  const tsData = resp && resp.data && resp.Data.find(d => d.time === timestampMS);
```

Comment:
This line should not be changed.

Reply:
You are right, it's just a mistake, I did undo that